pkgs/games/dwarf-fortress/wrapper/default.nix

```diff
@@ -16,46 +17,88 @@
 , twbt
 , themes ? { }
 , theme ? null
+, extraPackages ? [ ]
   # General config options:
 , enableIntro ? true
-, enableTruetype ? true
+, enableTruetype ? null # defaults to 24, see init.txt
```

Comment:
Note that previously the `enableTruetype` setting had no effect, because the configuration builder looked for `[TRUETYPE:YES]` instead of `[TRUETYPE:24]`, so we are free to pick a new default (I picked `null`, which leaves the default value 24 unchanged).

The new configuration mechanism works regardless of the default setting.